pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -349,6 +352,8 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance) {
 			params.MigrateDisks = copyDisks
 			params.MigrateDisksSet = true
 		}
+		// start live migration tracking
+		go liveMigrationMonitor(vmi, dom, l)
```

Comment:
I think there might be a race condition here. We're starting the tracking before the migration kicks off.

There's this code in the migration monitor go routine fails if the job can't be found, so it seems like there might be a situation where tracking fails depending on whether or not the migration kicked off in time. 

```
		stats, err := dom.GetJobInfo()
		if err != nil {
			logger.Reason(err).Error("failed to get domain job info")
			break
		}
```

Reply:
That shouldn't be the case, from what I've tested.
`GetJobInfo` should only return an exception if there is an issue with the domain or libvirt.
If there are no active jobs for this domain, it should just return `VIR_DOMAIN_JOB_NONE` and we don't break on that.
But, I'll double check in any case

Reply:
ah excellent, if that's the case then we're fine